start-console.sh

```diff
@@ -1,50 +1,74 @@
-#!/usr/bin/env bash
-#Please run this script under bash 4+
-set -euo pipefail
+#!/bin/sh
+set -eu
 
-#Cloning, pulling and running other plugins starting from port 9002
-# to add more plugin simply add more properites to dic. [name-of-plugin]={git-repo-url}
-declare -A plugins=(["monitoring-plugin"]="https://github.com/openshift/monitoring-plugin.git", ["networking-console-plugin"]="https://github.com/openshift/networking-console-plugin.git")
-declare -A runningPlugins=(["podman-linux"]="kubevirt-plugin=http://localhost:9001", ["podman"]="kubevirt-plugin=http://host.containers.internal:9001", ["docker"]="kubevirt-plugin=http://host.docker.internal:9001")
+# Define plugins (name=url)
+plugins="
+monitoring-plugin=https://github.com/openshift/monitoring-plugin.git
+networking-console-plugin=https://github.com/openshift/networking-console-plugin.git
+"
+
+# Plugin URLs per container runtime
+running_podman_linux="kubevirt-plugin=http://localhost:9001"
+running_podman="kubevirt-plugin=http://host.containers.internal:9001"
+running_docker="kubevirt-plugin=http://host.docker.internal:9001"
+
+get_plugin_url() {
+    name="$1"
+    echo "$plugins" | while IFS='=' read -r key val; do
+        [ "$key" = "$name" ] && echo "$val" && return
+    done
+}
 
 INITIAL_PORT=9002
-for arg in $@; do
+
+for arg in "$@"; do
+    plugin_url=$(get_plugin_url "$arg")
+
+    if [ -z "$plugin_url" ]; then
+        echo "Unknown plugin: $arg"
+        exit 1
+    fi
 
     if [ ! -d "../$arg" ]; then
-        echo "Creating a folder $arg ..."
-        cd ../
-        echo "Cloning "${plugins[$arg]}" ..."
-        git clone ${plugins[$arg]}
-        cd $arg
+        echo "Creating folder $arg ..."
+        cd ..
+        echo "Cloning $plugin_url ..."
+        git clone "$plugin_url"
+        cd "$arg"
     else
-        cd ../$arg
+        cd "../$arg"
     fi
+
     git pull
 
-    if [ "$(lsof -t -i:$INITIAL_PORT)" != "" ]; then
-        kill -9 $(lsof -t -i:$INITIAL_PORT)
-    fi
 
-    if [ $arg = "monitoring-plugin" ]; then
+    pid="$(lsof -t -i:$INITIAL_PORT 2>/dev/null || true)"
+    [ -n "$pid" ] && kill -9 "$pid"
+
+
+    [ "$arg" = "monitoring-plugin" ] then
```

Comment:
This line needs to be changed back to:

`if [ "$arg" = "monitoring-plugin" ]; then`